dvc/logger.py

```diff
@@ -66,18 +67,14 @@ def format(self, record):
             exception, stack_trace = self._parse_exc(record.exc_info)
 
             return (
-                "{color}{levelname}{nc}: {description}"
-                "{stack_trace}\n"
-                "\n"
-                "{footer}"
+                "{color}{levelname}{nc}: {description}" "{stack_trace}\n"
             ).format(
                 color=self.color_code.get(record.levelname, ""),
                 nc=colorama.Fore.RESET,
                 levelname=record.levelname,
                 description=self._description(record.msg, exception),
                 msg=record.msg,
```

Comment:
I see that deepsource detected that `msg` is not used here although I did not delete it
I think adding it back to message or removing it entirely should be scope of another PR to be honest, as it affects multiple test cases. 
What do you think @efiop?

Reply:
You are right, it is out of scope for this PR.